src/Workspace/Wes_Timing_System_4/_Config.lua

```diff
@@ -22,6 +22,8 @@ local config = {
 	CornerCutCooldownSeconds = 1,
 	-- Whether or not a popup appears per cut, rather than once per lap
 	PopupEverycut = false,
+	-- Whether or not the name displayed on the leaderboard is the username or display name
+	ShowDisplayNames = false,
```

Comment:
Note that this changes the default behaviour of the timing system from using display names to user names.